RELEASE.md

```diff
@@ -1,13 +1,4 @@
-# Upcoming release:
-
-<!--
-Use the sections below to add notes for the next release.
-
-Please follow the established format:
-- Keep each note concise - ideally commit title length
-- Use present tense (e.g. 'Add new feature')
-- Include the ID number for the related PR (or PRs) in parentheses
--->
+# Release 3.12.0
```

Comment:
I noticed that the release notes have been merged in a separate PR, which I had further supplemented with some comments. 

Think it would be nicer to keep the updates to the release notes with the release as one PR for better clarity?

Reply:
Yea sorry that was my mistake. I was following the process on core. Will be more careful next time.